recipes/duckdb-cli/meta.yaml

```diff
@@ -0,0 +1,114 @@
+{% set name = "duckdb-cli" %}
+{% set version = "0.9.1" %}
+{% set extensions = "httpfs;json;parquet" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/duckdb/duckdb/archive/v{{ version }}.tar.gz
+  sha256: 37a43188d9354ce3ca101b2b118d867f5f76d04c3b83c09d86fd7508351a631b
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - |
+      # echo "duckdb_extension_load(httpfs)" > extension/extension_config_local.cmake
+      EXTENSION_STATIC_BUILD=1 make extension_configuration
+      DUCKDB_EXTENSIONS='{{ extensions }}' VCPKG_TOOLCHAIN_PATH="$VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake" make
+      cp build/release/duckdb $PREFIX/bin
+      cp -R build/release/extension $PREFIX/
+  missing_dso_whitelist:
+    - '*/libpthread.so.0'
+    - '*/libdl.so.2'
+    - '*/libm.so.6'
+    - '*/libc.so.6'
+    - '*/librt.so.1'
+    - '*/ld-linux-x86-64.so.2'
+    - '*/libgcc_s.so.1'
+    - '*/libssl.so.3'
+    - '*/libcrypto.so.3'
+    - '*/libssl.3.dylib'
+    - '*/libcrypto.3.dylib'
+    - '*/libc++.1.dylib'
+    - '*/libstdc++.so.6'
```

Comment:
These should not be listed as this indicates missing dependencies in `host` or `build`.
```suggestion
```